express/code/blocks/faqv2/faqv2.css

```diff
@@ -0,0 +1,254 @@
+.faqv2-wrapper {
+  margin: 0 auto 10px auto;
+  box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+  border-radius: 8px;
+  background: var(--color-white);
+  width: 92%;
+}
+
+.faqv2-accordion {
+  height: 128px;
+  margin-bottom: 4px;
+  border-radius: 0px 8px 8px 0px;
+  border-left: 4px solid #724CDC;
+  background: #F8F8F8;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.faqv2 {
+  position: relative;
+  z-index: 1;
+  width: 100%;
+  height: auto;
+}
+
+.faqv2 .faqv2-accordion {
+  margin-right: 15px;
+}
+
+.faqv2-header {
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  padding-left: 16px;
+  color: #444444;
+}
+
+.faqv2-sub-header {
+  margin-top: 10px;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+  text-align: left;
+  padding-left: 16px;
+  color: #505050;
+}
+
+.faqv2 .faqv2-toggle-btn {
+  display: block;
+  width: fit-content;
+  margin: 15px 0 0 15px;
+  text-align: left;
+  text-decoration: none;
+  border-radius: 18px;
+  padding: 5px 1.2em 6px 1.2em;
+  font-size: var(--body-font-size-s);
+  font-weight: 600;
+  line-height: var(--body-line-height);
+  cursor: pointer;
+  transition: background-color 0.3s, color 0.3s, border 0.3s;
+  border-width: 2px;
+  border-style: solid;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  color: var(--color-white);
+  background-color: var(--color-info-accent);
+  border-color: var(--color-info-accent);
+}
+
+.faqv2-accordion.expandable.header-accordion {
+  height: 100%;
+  border-radius: 8px;
+  border: none;
+  margin-bottom: 6px;
+  background: white;
+}
+
+.faqv2-accordion.expandable.header-accordion.rounded-corners {
+  border-radius: 8px 8px 0 0;
+  margin-bottom: 0;
+  background: linear-gradient(to bottom,
+      #5C5CE0,
+      #9C49C6,
+      #BF38A5) left / 1% 100% no-repeat,
+    white;
+}
+
+.faqv2-accordion.expandable.sub-header-accordion {
+  height: 100%;
+  border-radius: 0 0 8px 8px;
+  border: none;
+  display: none;
+  margin-bottom: 6px;
+  background: linear-gradient(to bottom,
+      #D13381,
+      #DE3E67,
+      #EC5153,
+      #F2693C,
+      #F08420) left / 1% 100% no-repeat,
+    white;
+}
+
+.faqv2-header.expandable {
+  font-size: 16px;
+  font-weight: 700;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  color: #242424;
+  padding-left: 6px;
+}
+
+.faqv2-accordion.expandable .toggle-icon {
+  width: 12px;
+  height: 12px;
+}
+
+.faqv2-sub-header.expandable {
+  font-size: 16px;
+  font-weight: 400;
+  margin-top: 0;
+  padding-bottom: 10px;
+  padding-left: 6px;
+}
+
+.faqv2-accordion.title {
+  font-size: 32px;
+  font-weight: 800;
+  margin: 20px 0 10px 0;
+  padding-left: 0;
+  height: 50px;
+  border: none;
+  background: transparent;
+  text-align: center;
+}
+
+.section.faqv2-grey-bg {
```

Comment:
you shouldn't be adding stuff to the section @jsandland . Backgrounds should be authored in the section metadata. 
As a rule, a block should never be influencing the section. It's a bit like an an web component. A block should never really know what's going on outside of it or be affecting things outside of it.